spacepy/datamanager.py

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-The datamanager classes and functions are useful for locating the correct
-data file for a particular day and manipulating data and subsets in a generic
-way.
+Tool for locating the correct data file for a particular day and manipulating
+data and subsets generically.
```

Comment:
For all of these, where we're fixing them in this PR, we should probably stick to the "one line summary, blank line, more info" format.

This I'd just say "Tools for manipulating paths, data, and subsets" -- I still don't have the "file for a day" functionality in there.